edi/models/edi_backend.py

```diff
@@ -323,7 +322,7 @@ def _check_output_exchange_sync(self, skip_send=False):
             len(new_records),
         )
         for rec in new_records:
-            self.generate_output(rec)
+            self.with_delay().exchange_generate(rec)
```

Comment:
The only thing that makes me crazy with this change is that in order to send a record, it must enter two times on the cron, one for generating and one for sending :upside_down_face:

Reply:
what do you mean? The cron will run once and schedule one job per each record.

Reply:
When the cron is launched he will check if a record is generated, and then he will create the jobs, then, it will check the pending to send, and will create the jobs. The jobs will not be executed until all has finished.
If we have a exchange record not generated, it will enter on the first task, but not on the second, as it is not generated yet (it will be once all has finished)

Reply:
as discussed: we don't care. At worse the record will be sent 1h later. If this is an issue you can tweak the cron to run every 15 or 30m.